.github/workflows/release.yml

```diff
@@ -0,0 +1,167 @@
+            then
+              echo "Do not push image inside fork."
+              exit 0
+          fi
+
+          TAG=${GITHUB_SHA}
+          RELEASE_TAG=${GITHUB_REF##*/}
+
+          echo ${DOCKERHUB_PASS} | docker login -u ${DOCKERHUB_USER} --password-stdin
+
+
+          docker pull gresearchdev/armada-server-dev:${TAG}
+          docker tag gresearchdev/armada-server-dev:${TAG} gresearchdev/armada-server:${RELEASE_TAG}
+          docker push gresearchdev/armada-server:${RELEASE_TAG}
+
+          docker pull gresearchdev/armada-executor-dev:${TAG}
+          docker tag gresearchdev/armada-executor-dev:${TAG} gresearchdev/armada-executor:${RELEASE_TAG}
+          docker push gresearchdev/armada-executor:${RELEASE_TAG}
+
+          docker pull gresearchdev/armada-armadactl-dev:${TAG}
+          docker tag gresearchdev/armada-armadactl-dev:${TAG} gresearchdev/armada-armadactl:${RELEASE_TAG}
+          docker push gresearchdev/armada-armadactl:${RELEASE_TAG}
+
+          docker pull gresearchdev/armada-lookout-dev:${TAG}
+          docker tag gresearchdev/armada-lookout-dev:${TAG} gresearchdev/armada-lookout:${RELEASE_TAG}
+          docker push gresearchdev/armada-lookout:${RELEASE_TAG}
+
+          docker pull gresearchdev/armada-binoculars-dev:${TAG}
+          docker tag gresearchdev/armada-binoculars-dev:${TAG} gresearchdev/armada-binoculars:${RELEASE_TAG}
+          docker push gresearchdev/armada-binoculars:${RELEASE_TAG}
+
+          rm -v /home/runner/.docker/config.json
+        working-directory: ${{ env.GOPATH }}/src/github.com/G-Research/armada
+
+  update-charts:
+    needs: [release]
+    name: Publish Armada charts
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Armada
+        uses: actions/checkout@v2
+        with:
+          path: ${{ env.GOPATH }}/src/github.com/G-Research/armada
+
+      - name: Clone chart repo, update version and image tag to match current $RELEASE_TAG
+        run: |
+          mkdir -p ${{ env.GOPATH }}/src/github.com/G-Research/charts
+          mkdir -p $HOME/.ssh && echo "dummy" > $HOME/.ssh/known_hosts
+          
+          echo "github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==" > $HOME/.ssh/known_hosts
+          eval "$(ssh-agent -s)"
+          echo -e "${{ secrets.ARMADA_CHART_UPDATE_KEY }}" | ssh-add - > /dev/null
+          git clone -q git@github.com:G-Research/charts.git charts/
+          cd charts/
+          git remote -v
+
+          RELEASE_TAG=${GITHUB_REF##*/}
+          echo release version is $RELEASE_TAG
+          find ../armada \( -name "Chart.yaml" -o -name "values.yaml" \) -exec sed -i s/LATEST/$RELEASE_TAG/ {} +
```

Comment:
@ljubon can we use `yq` here maybe to replace contents of the file?

Reply:
at the moment, (code as it is) for `yq` we will need two lines for each file (`Chart.yaml` and `values.yaml`) as we have 6 different folders that mean instead of this one line we will need 12 `yq` lines :/

Reply:
still, valid comment, let me know what do you think (with the latest change)

Reply:
sounds good, let's leave it like this. i am not sure if `yq` can do a one-liner over a set of files. might be useful to investigate for future use, but not needed for this PR

Reply:
with latest findings we would need something like this
`yq eval-all -i '(.version = "CHANGEME"),(.image.tag = "CHANGEME"),(.appVersion = "CHANGEME")' ./deployment/armada/values.yaml ./deployment/armada/Chart.yaml`
but that's currently not supported, as [union](https://mikefarah.gitbook.io/yq/operators/union) + [eval-all](https://mikefarah.gitbook.io/yq/commands/evaluate-all#usage) doesn't seems to work toghether as expected :/

Reply:
let's see if we get any feedback from this https://github.com/mikefarah/yq/issues/453#issuecomment-1059723107

Reply:
FYI https://github.com/mikefarah/yq/issues/453